jobs/future-effective-filings/Dockerfile

```diff
@@ -1,35 +1,80 @@
-# platform=linux/amd64
-FROM python:3.8.6-buster
+FROM python:3.11.2-bullseye AS development_build
+
+USER root
 
 ARG VCS_REF="missing"
 ARG BUILD_DATE="missing"
 
 ENV VCS_REF=${VCS_REF}
 ENV BUILD_DATE=${BUILD_DATE}
+ENV PORT=8080
 
 LABEL org.label-schema.vcs-ref=${VCS_REF} \
     org.label-schema.build-date=${BUILD_DATE}
 
-USER root
+LABEL maintainer="karimjazzar"
```

Comment:
@thorwolpert I was looking at other Dockerfiles and I can see your name as the maintainer. Shall I put your name here instead of mine?

Reply:
nope, that's all good.